software/controller/test/commfail_alarm/commfail_alarm_test.cpp

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2020, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "commfail_alarm.h"
+#include "gtest/gtest.h"
+#include <cmath>
+#include <tuple>
```

Comment:
I don't see any need for cmath or tuple in the tests.

Reply:
removed not needed lines